scripts/dappnode_install_pre.sh

```diff
@@ -208,6 +246,13 @@ else
     install_lsof 2>&1 | tee -a $LOG_FILE
 fi
 
+# Only install wifi firmware if it does not detect the wifi card (Intel NUC 12)
+if [[ -z $(iw dev) ]]; then
```

Comment:
There is no check that device is indeed NUC gen 12 and that it has WIFI card.

Reply:
This has been solved in https://github.com/dappnode/DAppNode/pull/505/commits/bd4ba9e4605fe787c621acf52388f65e52b6ea04